04 Event Emitters/assets/js/app/App.js

```diff
@@ -0,0 +1,8 @@
+(function (window) {
```

Comment:
If you use `'use strict';` outside and IIFE all scripts below will automatically become strict-mode. If you use some non-strict written libraries below and they use not-allowed strict-mode methods it would throw an error (this is not the case as jQuery, React and ReactDOM uses strict-mode). I'm used to write all code inside an IIFE to encapsulate some vars and expose the public ones later as a practice so I use strict-mode inside the closure. What do you think?

BTW your third sentence will throw an error.